osf_tests/test_user.py

```diff
@@ -387,6 +399,76 @@ def test_merge_preprints(self, user):
         assert not preprint_five.has_permission(user2, permissions.READ)
         assert not preprint_five.is_contributor(user2)
 
+    def test_merge_drafts(self, user):
```

Comment:
I know this is consistent with other tests in this file, but consider breaking this up. Also the fact the `user2` is designated using the numeral, while `draft_two` is spelled out bothers me more then it probably should.